src/utils/BookingHelper.tsx

```diff
@@ -0,0 +1,59 @@
+import moment from "moment";
+import PositionDataFetcher from "./PositionDataFetcher";
+
+const fetchPositionData = async () => {
+    const apiEndpoint = "https://cc.vatsim-scandinavia.org/api/positions";
+
+    try {
+        const data = await PositionDataFetcher.fetchPositionData(apiEndpoint);
+        return data;
```

Comment:
```suggestion
        return await PositionDataFetcher.fetchPositionData(apiEndpoint);
```

Reply:
Corrected in commit [296cf95](https://github.com/Vatsim-Scandinavia/home/pull/32/commits/296cf958c8223de9253859276b0bb3daf17a78a9)